controlers/auth.js

```diff
@@ -98,9 +105,36 @@ const currentUser = (req, res) => {
     });
 };
 
+const avatarUpdate = async (req, res, next) => {
+    try {
+        if (!req.file) {
+            return res.status(400).json({ message: "No file uploaded" });
+        };
+        const storeImageDir = path.join(process.cwd(), "public/avatars");
+        const { path: tempPath } = req.file;
+        const extension = path.extname(tempPath);
+        const fileName = `${uuidV4()}${extension}`;
```

Comment:
Robisz już to w konfiguracji multer, sprawdzałaś czy to działa? Jeśli tak możnaby tamtego użyć tutaj.